src/Joueur.java

```diff
@@ -225,14 +220,7 @@ public String getName()
 	 */
 	public String toString()
 	{
-		switch(language)
```

Comment:
Why have you delete the basic multi-language support ? It is a bug or something ?